indexing-service/src/main/java/org/apache/druid/indexing/scheduledbatch/ScheduledBatchTaskManager.java

```diff
@@ -290,9 +334,9 @@ private Duration getTimeUntilNextTaskSubmission()
       return cronSchedulerConfig.getDurationUntilNextTaskStartTimeAfter(DateTimes.nowUtc());
     }
 
-    private ScheduledBatchSupervisorSnapshot.BatchSupervisorStatus getSchedulerStatus()
+    private ScheduledBatchSupervisor.State getSchedulerState()
```

Comment:
Since we are not using the `scheduler` name anymore.
```suggestion
    private ScheduledBatchSupervisor.State getState()
```